packages/isomorphic/test/utils.test.ts

```diff
@@ -23,8 +23,12 @@ describe('utils', function () {
     expect(hexToBytes('DEADBEEF')).toEqual(new Uint8Array([222, 173, 190, 239]))
   })
 
-  it('bytesToHex - DEADBEEF', () => {
-    expect(bytesToHex([222, 173, 190, 239])).toEqual('DEADBEEF')
+  it('hexToBytes - DEADBEEF', () => {
+    expect(hexToBytes('DEADBEEF')).toEqual(new Uint8Array([222, 173, 190, 239]))
+  })
+
+  it('bytesToHex - bad hex', () => {
+    expect(() => hexToBytes('hello')).toThrow(new Error('Invalid hex string'))
```

Comment:
Should this be a special Error that can be more easily detected?

Reply:
this is in the unit tests, so I don't expect a special error code to be useful here. On the other hand, I feel the main code base could make use of a special error code / error status message

Reply:
Yeah that is what I mean.

Reply:
I checked what errors were used in the rest of the package and it was just `Error`, so I just copied the norm

Reply:
hmm. Assigning special error codes (for all errors) might be a significant deviation from the intent of the PR. I'm okay if it's not included as a part of this change list